app/src/main/java/com/example/criminalintentapp/CrimeFragment.kt

```diff
@@ -31,9 +37,25 @@ class CrimeFragment : Fragment() {
             text = crime.date.toString()
         }
 
+        val crimeId: UUID = arguments?.getSerializable(ARG_CRIME_ID) as UUID
+        crimeDetailViewModel.loadCrime(crimeId)
+
         return view
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        crimeDetailViewModel.crimeLiveData.observe(
+            viewLifecycleOwner,
+            Observer { crime ->
```

Comment:
There's a Redundant SAM-constructor warning here, it's an easy fix with option + enter